packages/forms/src/widgets/DatalistWidget/DatalistWidget.js

```diff
@@ -159,7 +160,20 @@ class DatalistWidget extends React.Component {
 		if (hasItems && Object.prototype.hasOwnProperty.call(itemsMap, value)) {
 			return itemsMap[value];
 		}
-		return value;
+		return value != null ? value : '';
```

Comment:
Unfortunately, I can't. If `value` is `null`, the default parameter is not taken in account : 

```javascript
➜  ui git:(master) ✗ node
> function yo(test = 'lo') { console.log('yo', test); };

> yo();
yolo

> yo(null);
yonull

> 
```

But I learned something !